DuckDuckGo/Localizable.xcstrings

```diff
@@ -55243,6 +55243,18 @@
         }
       }
     },
+    "remove.suggestion.tooltip" : {
+      "comment" : "Tooltip for the button which removes the history entry from the history",
+      "extractionState" : "extracted_with_value",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "new",
+            "value" : "Remove from browsing history"
+          }
+        }
+      }
+    },
```

Comment:
Is this change intentional? (It may go away after merging `main` into this branch)

Reply:
Nooooo good catch.